inst/extdata/gcam-china/province_names_mappings.csv

```diff
@@ -0,0 +1,38 @@
+# File: province_names_mappings.csv
+# Title: Province alternate names mapping file
+# Units: NA
+# Description: Map province alternate names, climate region and grid region
```

Comment:
# Description: Map province alternate names, administrative region, climate region, and grid region